src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -280,6 +280,77 @@ Below is a list of available validate commands:
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
+### `gather`
+
+The `subctl gather` is a data collection tool that gathers Submariner/Kubernetes resources from all clusters in a cluster set by iterating
+over all kubeconfigs. The resources collected are:
+
+* pod logs for
```

Comment:
I don't think we need an exhaustive list of what's collected - I think the brief summary above suffices.

Reply:
Agree. This is a user guide and focus should be on letting user know how to use it, not what it is doing under the hood.